cpu/cc2538/clock.c

```diff
@@ -62,7 +62,21 @@
 #include <stdint.h>
 /*---------------------------------------------------------------------------*/
 #define RTIMER_CLOCK_TICK_RATIO (RTIMER_SECOND / CLOCK_SECOND)
-#define RELOAD_VALUE (125000 - 1)  /** Fire 128 times / sec */
+
+/* Prescaler for GPT0:Timer A used for clock_delay_usec(). */
+#if SYS_CTRL_SYS_CLOCK < SYS_CTRL_1MHZ
+#error System clock speeds below 1MHz are not supported
+#else
+#define PRESCALER_VALUE     (SYS_CTRL_SYS_CLOCK / 1000000 - 1)
```

Comment:
You could even use `SYS_CTRL_1MHZ` instead of `1000000` here to better match the test above.